engine/src/main/java/org/hibernate/search/engine/metadata/impl/BridgeDefinedField.java

```diff
@@ -57,4 +67,12 @@ public FieldType getType() {
 	public String toString() {
 		return "BridgeDefinedField [name=" + absoluteName + ", type=" + type + "]";
 	}
+
+	public void add(Class class1, Object elasticsearch) {
```

Comment:
This could use generics, so that we ensure (more or less) that the cast in `mappedOn` (below) will always work.

```
public <T> void add(Class<T> class1, T elasticsearch) {
  extra.put( class1, elasticsearch );
}
```

Also, "elasticsearch" should probably be renamed :)